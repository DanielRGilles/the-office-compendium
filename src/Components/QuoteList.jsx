import React from 'react'
import QuoteItem from './QuoteItem'
export default function QuoteList({quotes}) {
    return (
        <ul>
            {quotes.map((quote) => { 
                return (<QuoteItem quote={quote}/>)} )}
        </ul>
    )
}

