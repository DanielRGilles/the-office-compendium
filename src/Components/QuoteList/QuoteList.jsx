import React from 'react'
import QuoteItem from '../QuoteItem/QuoteItem'

export default function QuoteList({quotes}) {
    return (
        <ul>
            { quotes.map((quote) => { 
                    return (<QuoteItem quote={quote}/>)} )    
            }
        </ul>
    )
}

