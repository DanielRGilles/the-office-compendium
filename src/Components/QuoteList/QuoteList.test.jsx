import { render } from '@testing-library/react'
import QuoteList from './QuoteList'
import { data } from '../../Backup/BackupData'


it('Should render controls', async () => {
  

const { container } = render(<QuoteList quotes={data}/>)
  expect(container).toMatchSnapshot()
})