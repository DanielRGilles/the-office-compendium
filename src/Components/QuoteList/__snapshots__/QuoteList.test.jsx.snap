// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render controls 1`] = `
<div>
  <ul
    aria-label="list"
  >
    <li
      class="quote-item"
    >
      <img
        alt="Michael"
        src="Assets/Michael.jpg"
      />
      <div
        class="quote-text"
      >
        Michael
         
        Scott
        : 
        If I had a gun with two bullets and I was in a room with Hitler, Bin Laden, and Toby, I would shoot Toby twice.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Michael"
        src="Assets/Michael.jpg"
      />
      <div
        class="quote-text"
      >
        Michael
         
        Scott
        : 
        Occasionally, I’ll hit somebody with my car. So sue me.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Michael"
        src="Assets/Michael.jpg"
      />
      <div
        class="quote-text"
      >
        Michael
         
        Scott
        : 
        Everybody stay calm! Stay f*&#!#% calm!
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Michael"
        src="Assets/Michael.jpg"
      />
      <div
        class="quote-text"
      >
        Michael
         
        Scott
        : 
        I. Declare. Bankruptcy!
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Michael"
        src="Assets/Michael.jpg"
      />
      <div
        class="quote-text"
      >
        Michael
         
        Scott
        : 
        And I knew exactly what to do. But in a much more real sense, I had no idea what to do.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Michael"
        src="Assets/Michael.jpg"
      />
      <div
        class="quote-text"
      >
        Michael
         
        Scott
        : 
        I feel like all my kids grew up, and then they married each other. It’s every parent’s dream.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Michael"
        src="Assets/Michael.jpg"
      />
      <div
        class="quote-text"
      >
        Michael
         
        Scott
        : 
        Dwight, you ignorant slut!
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Michael"
        src="Assets/Michael.jpg"
      />
      <div
        class="quote-text"
      >
        Michael
         
        Scott
        : 
        Well, Happy Birthday Jesus. Sorry your party’s so lame.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Michael"
        src="Assets/Michael.jpg"
      />
      <div
        class="quote-text"
      >
        Michael
         
        Scott
        : 
        Sometimes I’ll start a sentence, and I don’t even know where it’s going. I just hope I find it along the way.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Michael"
        src="Assets/Michael.jpg"
      />
      <div
        class="quote-text"
      >
        Michael
         
        Scott
        : 
        There are certain things a boss does not share with his employees. His salary, that would depress them. His bed, it--- And I am not going to tell them that I'll be reading their e-mails.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Pam"
        src="Assets/Pam.jpg"
      />
      <div
        class="quote-text"
      >
        Pam
         
        Beesly
        : 
        Once every hour, someone is involved in an internet scam. That man is Michael Scott.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Pam"
        src="Assets/Pam.jpg"
      />
      <div
        class="quote-text"
      >
        Pam
         
        Beesly
        : 
        It's performance review day, company-wide. Last year, my performance review started with Michael asking me what my hopes and dreams were, and it ended with him telling me he could bench-press 190 pounds. So, I don't really know what to expect.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Pam"
        src="Assets/Pam.jpg"
      />
      <div
        class="quote-text"
      >
        Pam
         
        Beesly
        : 
        I feel God in this Chili’s tonight.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Pam"
        src="Assets/Pam.jpg"
      />
      <div
        class="quote-text"
      >
        Pam
         
        Beesly
        : 
        There’s a lot of beauty in ordinary things. Isn’t that kind of the point?
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Creed"
        src="Assets/Creed.jpg"
      />
      <div
        class="quote-text"
      >
        Creed
         
        Bratton
        : 
        Two eyes. Two ears. A chin. A mouth. Ten fingers. Two nipples. A butt, two kneecaps, a penis. I've just described to you the Loch Ness monster. And the reward for his capture? All the riches in Scotland. So I have one question: Why are you here?
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Kevin"
        src="Assets/Kevin.jpg"
      />
      <div
        class="quote-text"
      >
        Kevin
         
        Malone
        : 
        I am immensely proud of what I did for that turtle!
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Stanley"
        src="Assets/Stanley.jpg"
      />
      <div
        class="quote-text"
      >
        Stanley
         
        Hudson
        : 
        I wake up every morning in a bed that's too small, drive my daughter to a school that's too expensive, and then I go to work to a job for which I get paid too little. But on pretzel day? Well, I like pretzel day.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Stanley"
        src="Assets/Stanley.jpg"
      />
      <div
        class="quote-text"
      >
        Stanley
         
        Hudson
        : 
        Did I stutter?
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Ryan"
        src="Assets/Ryan.jpg"
      />
      <div
        class="quote-text"
      >
        Ryan
         
        Howard
        : 
        Last year, Creed asked me how to set up a blog. Wanting to protect the world from being exposed to Creed's brain, I opened up a Word document on his computer and put an address at the top. I've read some of it. Even for the Internet, it's... pretty shocking.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Kelly"
        src="Assets/Kelly.jpg"
      />
      <div
        class="quote-text"
      >
        Kelly
         
        Kapoor
        : 
        I don't talk trash, I talk smack. They're totally different. Trash talk is hypothetical, like: Your mom is so fat she can eat the internet. But smack talk is happening like right now. Like: You're ugly and I know it for a fact 'cause I got the evidence right there.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Kelly"
        src="Assets/Kelly.jpg"
      />
      <div
        class="quote-text"
      >
        Kelly
         
        Kapoor
        : 
        I have a lot of questions. Number one, how dare you?
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Jim"
        src="Assets/Jim.jpg"
      />
      <div
        class="quote-text"
      >
        Jim
         
        Halpert
        : 
        I had feelings today for a coworker that I haven't had in years... In my defense he was grabbing my crotch fairly aggressively at the time.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Jim"
        src="Assets/Jim.jpg"
      />
      <div
        class="quote-text"
      >
        Jim
         
        Halpert
        : 
        Bears, Beets, Battlestar Galactica.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Jim"
        src="Assets/Jim.jpg"
      />
      <div
        class="quote-text"
      >
        Jim
         
        Halpert
        : 
        It's true. I'm having a party. I've got three cases of imported beer, a karaoke machine, and I didn't invite Michael. So three ingredients for a great party. And it's nothing personal, I just think that if he were there, people wouldn't be able to relax, and you know, have fun, and my roommate wants to meet everybody. Because I'm pretty sure he thinks that I'm making Dwight up. [sighs] He is very real.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Andy"
        src="Assets/Andy.jpg"
      />
      <div
        class="quote-text"
      >
        Andy
         
        Bernard
        : 
        I wish there was a way to know you’re in the good old days, before you’ve actually left them.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Andy"
        src="Assets/Andy.jpg"
      />
      <div
        class="quote-text"
      >
        Andy
         
        Bernard
        : 
        You can't let a girl feel good about herself. It will backfire on you. Every compliment has to be backhanded. 'Oh I like your dress, but I'd like it more if you had prettier hair.'
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Andy"
        src="Assets/Andy.jpg"
      />
      <div
        class="quote-text"
      >
        Andy
         
        Bernard
        : 
        I'll be the Number Two guy here in Scranton in six weeks. How? Name repetition, personality mirroring, and never breaking off a handshake. I'm always thinking one step ahead. Like a carpenter that makes stairs.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Andy"
        src="Assets/Andy.jpg"
      />
      <div
        class="quote-text"
      >
        Andy
         
        Bernard
        : 
        Sorry I annoyed you with my friendship.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Dwight"
        src="Assets/Dwight.jpg"
      />
      <div
        class="quote-text"
      >
        Dwight
         
        Schrute
        : 
        Whenever I'm about to do something, I think, 'Would an idiot do that?' And if they would, I do not do that thing.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Dwight"
        src="Assets/Dwight.jpg"
      />
      <div
        class="quote-text"
      >
        Dwight
         
        Schrute
        : 
        It has to be official, and it has to be urine.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Dwight"
        src="Assets/Dwight.jpg"
      />
      <div
        class="quote-text"
      >
        Dwight
         
        Schrute
        : 
        I have been Michael's #2 guy for about 5 years. And we make a great team. We're like one of those classic famous teams. He's like Mozart, and I'm like Mozart's friend. No. I'm like Butch Cassidy, and Michael is like Mozart. You try and hurt Mozart; you're going to get a bullet in your head courtesy of Butch Cassidy.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Dwight"
        src="Assets/Dwight.jpg"
      />
      <div
        class="quote-text"
      >
        Dwight
         
        Schrute
        : 
        I.D. badges are long overdue. Security in this office park is a joke. Last year, I came to work with my spud gun in a duffel bag. I sat at my desk all day, with a rifle that shoots potatoes at 60 pounds per square inch. Can you imagine if I was deranged?
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Angela"
        src="Assets/Angela.jpg"
      />
      <div
        class="quote-text"
      >
        Angela
         
        Martin
        : 
        If you pray enough, you can change yourself into a cat person.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Angela"
        src="Assets/Angela.jpg"
      />
      <div
        class="quote-text"
      >
        Angela
         
        Martin
        : 
        I know that patience and loyalty are good, and virtuous traits. But sometimes I just think you need to grow a pair.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Oscar"
        src="Assets/Oscar.jpg"
      />
      <div
        class="quote-text"
      >
        Oscar
         
        Martinez
        : 
        I didn't realize how many of Angela's opinions I agreed with...until she tried to have my knee caps shattered for sleeping with her husband.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Phyllis"
        src="Assets/Phyllis.jpg"
      />
      <div
        class="quote-text"
      >
        Phyllis
         
        Lapin
        : 
        I'm glad Michael's getting help. He has a lot of issues, and he's stupid.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Toby"
        src="Assets/Toby.jpg"
      />
      <div
        class="quote-text"
      >
        Toby
         
        Flenderson
        : 
        Chad Flenderman. Just an easy-going black guy who knows the streets--yet he also went to Oxford. So. Just as comfortable on a motorcycle as he is on Air Force One. Oh and he's also the world's leading Egyptologist.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Erin"
        src="Assets/Erin.jpg"
      />
      <div
        class="quote-text"
      >
        Erin
         
        Hannon
        : 
        Disposable cameras are fun, but it seems a little wasteful. You never get to see your pictures. If it's an important event that you want to remember, I recommend using a real camera.
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Gabe"
        src="Assets/Gabe.jpg"
      />
      <div
        class="quote-text"
      >
        Gabe
         
        Lewis
        : 
        Shut up about the sun. SHUT UP ABOUT THE SUN!
      </div>
    </li>
    <li
      class="quote-item"
    >
      <img
        alt="Darryl"
        src="Assets/Darryl.jpg"
      />
      <div
        class="quote-text"
      >
        Darryl
         
        Philbin
        : 
        In a gang world, we use something called fluffy fingers. That's when someone really gets in your face, you know you just, start ticklin' 'em. And he starts tickling you. And pretty soon you laughing and hugging. Before you know it, you've forgotten the whole thing. Y'all just go to church together, and get an ice cream cone.
      </div>
    </li>
  </ul>
</div>
`;
